rules-java-archives/addon/src/main/java/org/jboss/windup/rules/apps/java/archives/identify/CompoundIdentifier.java

```diff
@@ -0,0 +1,34 @@
+package org.jboss.windup.rules.apps.java.archives.identify;
+
+import java.util.LinkedHashSet;
+import java.util.Set;
+import org.jboss.forge.addon.dependencies.Coordinate;
+import org.jboss.windup.rules.apps.java.archives.identify.api.ArchiveHashIdentifier;
+
+/**
+ *
+ *  @author Ondrej Zizka, ozizka at redhat.com
+ */
+public class CompoundIdentifier implements ArchiveHashIdentifier
```

Comment:
'Composite' is the word you're looking for here, I think (instead of Compound). They are similar but typically you see Composite used more frequently in this context.  (And 'Composite' is the term we've used previously.)